src/envo/envo-edit.owl

```diff
@@ -40852,7 +40852,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501165> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_03501166> (ophthalmology unit facility)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501166> "A hospital unit facility which is used to provide opthalmological medicine."@en)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501166> "A hospital unit facility which is used to provide opthalmological medicine."@en)
```

Comment:
@ddooley Do you want the definition to say "provide ophthalmological services" instead of "provide opthalmological medicine"? Also is "opthalmological" spelled wrong? Should it be "ophthalmological"?

Reply:
suggestion and spelling accepted!